course_discovery/apps/course_metadata/management/commands/add_logos_to_organization.py

```diff
@@ -0,0 +1,46 @@
+""" Command to add logo, certificate logo, and banner image to an organization """
+from django.core.files import File
+from django.core.management.base import BaseCommand, CommandError
+
+from course_discovery.apps.core.models import Partner
+from course_discovery.apps.course_metadata.models import Organization
+
+
+class Command(BaseCommand):
+    """
+    Uploads given images to an organization. This command is meant to be run
+    from a provisioning script and not by hand.
+    """
+    def add_arguments(self, parser):
+        parser.add_argument('--partner', default='edX')
+        parser.add_argument('--logo', default='/edx/app/discovery/discovery/provision-temp/assets/demo-asset-logo.png')
+        parser.add_argument('--certificate_logo', default='/edx/app/discovery/discovery/provision-temp/assets/demo-asset-certificate-logo.png')
+        parser.add_argument('--banner_image', default='/edx/app/discovery/discovery/provision-temp/assets/demo-asset-banner-image.png')
+
+    def handle(self, *args, **kwargs):
+        partner_name = kwargs.get('partner')
+        logo_path = kwargs.get("logo")
+        certificate_logo_path = kwargs.get('certificate_logo')
+        banner_image_path = kwargs.get('banner_image')
+        try:
+            organization = Organization.objects.get(partner=self._get_partner_by_name(partner_name))
```

Comment:
I guess no need to make an extra query for the partner and instead, you can directly do this:
```suggestion
            organization = Organization.objects.get(partner__name=partner_name)
```

Reply:
Thanks for the suggestion. 🔧